Automattic/woocommerce-payments
# Add checks to prevent interacting with multi-currency if it is disabled

Fixes Automattic/woocommerce-payments-server#3209

#### Changes proposed in this Pull Request









#### Testing instructions





* Don't checkout to this branch yet
* Run `npm run start`
* Go to **WooCommerce > Settings** and change the store currency for any other than USD, and save changes
* Go to **Payments > Settings**
* Click on "Advanced settings", uncheck "Enable Multi-Currency", click on "Save changes"
* Refresh the page
* You should see a notice "Error retrieving currencies."
* Open the standard modal
* It should contain "An order exceeds the equivalent of $1,000.00 USD in undefined."
* **Checkout to this branch**
* Refresh the page
* There should be no notice
* Open the same modal – it should contain "An order exceeds the equivalent of $1,000.00 USD"
* The unit tests should pass

-------------------


**Post merge**
